cluster/charts/universal-crossplane/values.yaml

```diff
@@ -165,61 +165,18 @@ extraVolumesCrossplane: {}
 # -- Add custom `volumeMounts` to the Crossplane pod.
 extraVolumeMountsCrossplane: {}
 
```

Comment:
Shout out to @phisco for automating the chart values/readme management 🙌 
It is great not to worry about that while updating to a new crossplane version.